relay-server/Cargo.toml

```diff
@@ -57,7 +57,7 @@ relay-general = { path = "../relay-general" }
 relay-log = { path = "../relay-log" }
 relay-quotas = { path = "../relay-quotas" }
 relay-redis = { path = "../relay-redis" }
-reqwest = { version = "0.11.1", features = ["gzip", "stream", "trust-dns"] }
+reqwest = { version = "0.11.1", features = ["gzip", "stream", "trust-dns", "native-tls-vendored"] }
```

Comment:
This is the relevant change: By vendoring `native-tls`, we include `openssl-src` and compile ourselves. 

The inverted cargo tree of the Relay binary shows that there is only a single `openssl-sys` in the entire code base:

```
openssl-sys v0.9.58
├── native-tls v0.2.7
│   ├── actix-net v0.2.6
│   │   └── actix-web v0.7.19
│   │       └── relay-server v21.2.0 (/Users/jauer/Coding/relay/relay-server)
│   │           └── relay v21.2.0 (/Users/jauer/Coding/relay/relay)
│   ├── actix-web v0.7.19 (*)
│   ├── hyper-tls v0.5.0
│   │   └── reqwest v0.11.1
│   │       ├── relay-server v21.2.0 (/Users/jauer/Coding/relay/relay-server) (*)
│   │       └── sentry v0.22.0
│   │           └── relay-log v21.2.0 (/Users/jauer/Coding/relay/relay-log)
│   │               ├── relay v21.2.0 (/Users/jauer/Coding/relay/relay)
│   │               ├── relay-common v21.2.0 (/Users/jauer/Coding/relay/relay-common)
│   │               │   ├── relay v21.2.0 (/Users/jauer/Coding/relay/relay)
│   │               │   ├── relay-auth v21.2.0 (/Users/jauer/Coding/relay/relay-auth)
│   │               │   │   ├── relay-config v21.2.0 (/Users/jauer/Coding/relay/relay-config)
│   │               │   │   │   ├── relay v21.2.0 (/Users/jauer/Coding/relay/relay)
│   │               │   │   │   └── relay-server v21.2.0 (/Users/jauer/Coding/relay/relay-server) (*)
│   │               │   │   └── relay-server v21.2.0 (/Users/jauer/Coding/relay/relay-server) (*)
│   │               │   ├── relay-config v21.2.0 (/Users/jauer/Coding/relay/relay-config) (*)
│   │               │   ├── relay-filter v21.2.0 (/Users/jauer/Coding/relay/relay-filter)
│   │               │   │   └── relay-server v21.2.0 (/Users/jauer/Coding/relay/relay-server) (*)
│   │               │   ├── relay-general v21.2.0 (/Users/jauer/Coding/relay/relay-general)
│   │               │   │   ├── relay-filter v21.2.0 (/Users/jauer/Coding/relay/relay-filter) (*)
│   │               │   │   └── relay-server v21.2.0 (/Users/jauer/Coding/relay/relay-server) (*)
│   │               │   ├── relay-quotas v21.2.0 (/Users/jauer/Coding/relay/relay-quotas)
│   │               │   │   └── relay-server v21.2.0 (/Users/jauer/Coding/relay/relay-server) (*)
│   │               │   └── relay-server v21.2.0 (/Users/jauer/Coding/relay/relay-server) (*)
│   │               ├── relay-config v21.2.0 (/Users/jauer/Coding/relay/relay-config) (*)
│   │               └── relay-server v21.2.0 (/Users/jauer/Coding/relay/relay-server) (*)
│   ├── relay-server v21.2.0 (/Users/jauer/Coding/relay/relay-server) (*)
│   ├── reqwest v0.11.1 (*)
│   ├── tokio-native-tls v0.3.0
│   │   ├── hyper-tls v0.5.0 (*)
│   │   └── reqwest v0.11.1 (*)
│   └── tokio-tls v0.2.1
│       └── actix-web v0.7.19 (*)
└── openssl v0.10.30
    └── native-tls v0.2.7 (*)
```